packages/salesforcedx-vscode-core/test/context/workspaceOrgType.test.ts

```diff
@@ -228,6 +257,28 @@ describe('setupWorkspaceOrgType', () => {
     authInfoCreateStub.restore();
     executeCommandStub.restore();
   });
+
+  it('should set both sfdx:default_username_has_change_tracking and sfdx:default_username_has_no_change_tracking contexts to false for cli config error', async () => {
```

Comment:
Won't this error be handled by the default switch case on lines 53 & 54 of workspaceOrgType https://github.com/forcedotcom/salesforcedx-vscode/pull/868/files#diff-6145aff21335781c022072ea6842d8d2R53, in which case `sfdx:default_username_has_change_tracking` and  `sfdx:default_username_has_no_change_tracking` will be set to true, not false?

Reply:
That's correct, not sure why this test is passing. Let me take a look at it.